.github/workflows/publish.yml

```diff
@@ -60,6 +60,7 @@ jobs:
     - name: Git push version up commits
       run: |
         git push origin ${{ github.ref_name }}
+        git push --tags
```

Comment:
```suggestion
        git push origin --tags ${{ github.ref_name }}
```

Reply:
Didn't know this notation! looks good

See: https://git-scm.com/docs/git-push#Documentation/git-push.txt---tags